lib/content-services/src/lib/tag/tags-creator/tags-creator.component.spec.ts

```diff
@@ -341,32 +315,36 @@ describe('TagsCreatorComponent', () => {
             fixture.detectChanges();
             tick(100);
 
-            clickAtHideNameInputButton();
             component.tagNameControlVisible = true;
             fixture.detectChanges();
             tick(100);
 
             expect(getNameInput()).toBe(document.activeElement as HTMLInputElement);
         }));
 
-        it('should be hidden and cleared on discard changes', fakeAsync(() => {
-            component.tagNameControlVisible = true;
-            component.tags = ['Passed tag 1', 'Passed tag 2'];
-            typeTag('test');
-            fixture.detectChanges();
-            tick(100);
-            expect(getNameInput().value).toBe('test');
-
-            component.tagNameControlVisible = false;
-            fixture.detectChanges();
-            tick(100);
-            expect(getNameInput()).toBeFalsy();
+        describe('isNameTagsVisible', () => {
+            it('should return true when tagNameControlVisible is true', () => {
```

Comment:
Please test if UI works correctly not only logical conditions

Reply:
I have moved the input parameter to the template and removed the method and test cases related to this.